lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -130,42 +136,44 @@ Technical details: ${details || 'N/A'}
   });
 
   return (
-    <div className="ErrorDisplay">
-      {children}
-      {description && (
-        <p data-testid="message">
-          {description}
-          {error.message && (
-            <>
-              : <i>{toSentence(error.message)}</i>
-            </>
-          )}
+    <Scrollbox classes="LMS-Scrollbox">
```

Comment:
This diff is a bit of a pain to read, sorry. The changes are:

* Add `Scrollbox` container
* Update classes applied to elements
* Update `data-testid` attribute values to be more specific